### PR TITLE
tests: fix test_cli_help

### PR DESCRIPTION
**Pull request by karolinavelkaja**
Signed-off-by: Karolina Rogowska <karolina.rogowska@intel.com>

### CODE REVIEW

**Review by pdebski21 — APPROVED** [review]

File: test/functional/tests/cli/test_cli_help_and_version.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-# Copyright(c) 2020-2021 Intel Corporation
+# Copyright(c) 2020-2022 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-import pytest
 import re
 
+import pytest
+
```
Comment (robertbaldyga) [inline test/functional/tests/cli/test_cli_help_and_version.py:9]: Why this change? Most of the tests import pytest before other imports. Do we want to change this convention?

Comment (karolinavelkaja) [reply test/functional/tests/cli/test_cli_help_and_version.py:9]: I suspect this is result of "optimize imports", I don't recall to do such changes, there was no need. I can revert this if it can make any difference - does it?

**Review by mmichal10 — APPROVED** [review]

**Review by pdebski21 — APPROVED** [review]
